test/geometry/GeoParams.test.cc

```diff
@@ -0,0 +1,39 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file GeoParams.test.cc
+//---------------------------------------------------------------------------//
+#include "geometry/GeoParamsTest.hh"
+
+// using namespace celeritas;
+using namespace celeritas_test;
+
+//---------------------------------------------------------------------------//
+// HOST TESTS
+//---------------------------------------------------------------------------//
+
+class GeoParamsHostTest : public GeoParamsTest
+{
+  public:
+    void SetUp() override
+    {
+        host_view = params()->host_view();
+        CHECK(host_view.world_volume);
+    }
+
+    // Views
+    GeoParamsPointers host_view;
```

Comment:
I think the "view" nomenclature here is carried over from some of the older code before we all agreed on a terminology (pointers/trackview/view) and was something I missed while changing the code. Could you take the opportunity to fix my error and make sure the names are consistent?

Reply:
It looks to me like in some places, e.g. client classes or unit testing, the view nomenclature is appropriate (params_view), so quite a bit subjective.